package.json

```diff
@@ -23,7 +23,7 @@
     "@testing-library/react": "^14.0.0",
     "@vitejs/plugin-react": "^4.0.0",
     "jsdom": "^22.0.0",
-    "rescript": "^10.1.4",
+    "rescript": "^11.0.0-alpha.6",
```

Comment:
Maybe want to try beta 👀 [1.0.0-beta.2](https://github.com/rescript-lang/rescript-compiler/releases/tag/v11.0.0-beta.2)

Reply:
Upgraded in https://github.com/jihchi/vitejs-template-react-rescript/pull/18/commits/a86cde10f89b920473eff7fe0fc94ee7e8f74f74

Reply:
@jihchi we can use `^11.0.1`